modules/tinymce/src/core/main/ts/keyboard/MatchKeys.ts

```diff
@@ -23,17 +23,15 @@ const baseKeyPattern = {
   keyCode: 0
 };
 
-const defaultPatterns = (patterns: KeyPattern[]): KeyPattern[] =>
+const defaultPatterns = (patterns: KeyPattern[]): Required<KeyPattern>[] =>
   Arr.map(patterns, (pattern) => ({
     ...baseKeyPattern,
-    action: Fun.noop,
```

Comment:
Removed this as `action` is mandatory for `KeyPattern` so this was never used and strict mode raises that as an issue.